install_config/aggregate_logging.adoc

```diff
@@ -47,6 +47,9 @@ securely.
 xref:../install_config/install/host_preparation.adoc#managing-docker-container-logs[Managing
 Docker Container Logs] discusses the use of `json-file` logging driver options
 to manage container logs and prevent filling node disks.
+
+If deploying aggregated logging, it is strongly recommended that you
```

Comment:
Can we make this "stronger" still?

    "Aggregated logging is only supported using the `journald` driver in Docker."

Or something like that?